rules/wiz_rules/wiz_alert_passthrough.py

```diff
@@ -0,0 +1,32 @@
+def rule(event):
+    return event.get("status") == "OPEN"
+
+
+def title(event):
+    return (
+        f"[Wiz Alert]: "
+        f"{event.deep_get('sourceRule', 'name', default='ALERT_DESCRIPTION_NOT_FOUND')}"
+    )
+
+
+def severity(event):
+    return event.get("severity")
```

Comment:
Does  Panther have a 1-1 correlation to the Wiz alerts? (`INFO`, `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`)

Reply:
Yes, it does. It's convenient :)